entc/gen/template/predicate.tmpl

```diff
@@ -17,7 +19,7 @@ import (
 	"entgo.io/ent/dialect/sql"
 )
 
-{{ range $_, $n := $.Nodes -}}
+{{ range $n := $.Nodes -}}
```

Comment:
hey @a8m, I believe this fix is wrong. See: https://play.golang.org/p/TXAOkWN43pp

<img width="436" alt="Screen Shot 2021-04-09 at 17 57 47" src="https://user-images.githubusercontent.com/12751435/114170454-29f71900-995d-11eb-863c-0c0855c76ddf.png">

Reply:
The range clause works a bit different in templates (see the `text/template` [doc](https://golang.org/pkg/text/template/#hdr-Variables)):

> Note that if there is only one variable, it is assigned the element; this is opposite to the convention in Go range clauses.

But I still appreciate the review. Thanks 🙏

Reply:
Yeah, I have checked the doc but missed the note. :)))